tests/Feature/MemberControllerTest.php

```diff
@@ -0,0 +1,875 @@
+<?php
+
+namespace Tests\Feature;
+
+use Tests\TestCase;
+use App\Member;
+use Faker\Factory;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Http\Response;
+use Lang;
+use File;
+
+class MemberControllerTest extends TestCase
+{
+    use DatabaseMigrations;
+    use WithoutMiddleware;
+    /**
+     * A basic test example.
+     *
+     * @return void
+     */
+    public function testCreateMemberHaveAvatar()
+    {
+        $stub = public_path().'/img/aaa.jpg';
+        $name = str_random(8).'.jpg';
+        $path = public_path().'/img/test/'.$name;
+        copy($stub, $path);
+        $file = new UploadedFile($path, $name, 'image/jpg', filesize($path), null, true);
+        $newMember = Factory(Member::class)->create([
+        'name' => 'thu',
+        'information' => 'inter',
+        'phone_number' => '0978716945',
+        'birthday' => '1996-12-12',
+        'position_id' => 1,
+        'gender' => 'female',
+        'avatar' => $file,
+        ])->toArray();
+        $response = $this->json('POST', 'members/create', $newMember);
+        $this->assertEquals(200, $response->status());
+        $this->assertDatabaseHas('members', [
+            'name' => $newMember['name'],
+            'information' => $newMember['information'],
+            'phone_number' => $newMember['phone_number'],
+            'birthday' => $newMember['birthday'],
+            'position_id' => $newMember['position_id'],
+            'gender' => $newMember['gender']
+            ]);
+    }
+
+    public function testCreateMemberHaveAvatarMorethan10MB()
+    {
+        $stub = public_path().'/img/nui.jpeg';
+        $name = str_random(8).'.jpeg';
+        $path = public_path().'/img/test/'.$name;
+        copy($stub, $path);
+        $file = new UploadedFile($path, $name, 'image/jpeg', filesize($path), null, true);
+        $newMember = [
+        'name' => 'thuuu',
+        'information' => 'interu',
+        'phone_number' => '09716945',
+        'birthday' => '1996-12-10',
+        'position_id' => 3,
+        'gender' => 'female',
+        'avatar' => $file,
+        ];
+        $response = $this->json('POST', 'members/create', $newMember);
```

Comment:
You should also verify the JSON response.

Reply:
because when i add name image into DB, I have create name image random  avoid same name, so if  i verify JSON , name file not same name file JSON response